test/model.test.js

```diff
@@ -196,65 +196,75 @@ describe('loopback:model generator', function() {
       }
     });
 
-    describe.skip('with --bluemix', function() {
+    describe('with --bluemix', function() {
       it('should not throw if no Bluemix datasources are found',
-        function(done) {
+        function() {
           var srcPath = path.join(__dirname, 'fixtures',
             'datasources-config-empty.json');
           var destPath = path.join(SANDBOX, '.bluemix',
             'datasources-config.json');
           fs.copySync(srcPath, destPath);
 
-          var modelGen = givenModelGenerator('--bluemix --login=false');
-          helpers.mockPrompt(modelGen, {
-            name: 'Product',
-          });
-
-          modelGen.run(function() {
-            expect(modelGen.abort).to.eql(true);
-            done();
-          });
+          var ctx = helpers.run(path.join(__dirname, '../model'));
+          return ctx.cd(SANDBOX)
+            .withPrompts({
+              name: 'Product',
+            })
+            .withOptions({
+              bluemix: true,
+              login: false,
+            })
+            .then(function() {
+              expect(ctx.generator.abort).to.equal(true);
+            });
         });
 
-      it('should not throw on parsing datasources-config.json', function(done) {
+      it('should not throw on parsing datasources-config.json', function() {
         var srcPath = path.join(__dirname, 'fixtures',
           'datasources-config-empty.json');
         var destPath = path.join(SANDBOX, '.bluemix',
           'datasources-config.json');
         fs.copySync(srcPath, destPath);
         fs.writeFileSync(destPath, '');
 
-        var modelGen = givenModelGenerator('--bluemix --login=false');
-        helpers.mockPrompt(modelGen, {
-          name: 'Product',
-        });
-
-        modelGen.run(function() {
-          expect(modelGen.abort).to.eql(true);
-          done();
-        });
+        var ctx = helpers.run(path.join(__dirname, '../model'));
```

Comment:
Use `const`, not `var` - see https://loopback.io/doc/en/contrib/style-guide-es6.html